### PR TITLE
added atrium+spek sample project under samples/jvm dir

### PR DESCRIPTION
**Pull request by iamshubh**
----
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

**Comment by iamshubh** [comment]
It is fresh PR, one dupliacted PR #242 has been closed

File: .github/workflows/java-windows.yml [hunk 1]
```diff
@@ -18,3 +18,5 @@ jobs:
                     java-version: ${{ matrix.java_version }}
             -   name: Build with Gradle
                 run: ./gr jvmBuild
+            -   name: build sample atrium+spek project
+                run: ./gr -p samples\jvm\spek build
```
Comment (robstoll) [inline .github/workflows/java-windows.yml]: Although this works as well we are not using the gradle version of the sample. Change to:
```suggestion
                run: samples\jvm\spek\gradlew -p samples\jvm\spek build
```

File: .travis.yml [hunk 2]
```diff
@@ -18,6 +18,9 @@ install: true
 
 script:
   - |
+    if [ $MODE == current ]; then
+      ./gr -p ./samples/jvm/spek build
```
Comment (robstoll) [inline .travis.yml]: Same here

File: .github/workflows/java-windows.yml (hunk 1 above)
Comment (iamshubh) [reply .github/workflows/java-windows.yml]: added

File: .travis.yml (hunk 2 above)
Comment (iamshubh) [reply .travis.yml]: changed

**Comment by robstoll** [comment]
@iamshubh thanks for the contribution :+1:
How about another one? For instance #81 is similar but feel free to do another one (I'll add more later on this evening):
https://github.com/robstoll/atrium/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest+no%3Aassignee

**Pull request merged**